locale/en/docs/guides/publishing-napi-modules.md

```diff
@@ -1,53 +1,57 @@
-## To publish N-API version of a package alongside a non-N-API version
+---
+title: How to publish N-API package
+layout: docs.hbs
+---
+
+# To publish N-API version of a package alongside a non-N-API version
 
 The following steps are illustrated using the package `iotivity-node`:
   - First, publish the non-N-API version:
-    - Update the version in `package.json`. For `iotivity-node`, the version 
+    - Update the version in `package.json`. For `iotivity-node`, the version
     becomes `1.2.0-2`.
     - Go through the release checklist (ensure tests/demos/docs are OK)
     - `npm publish`
   - Then, publish the N-API version:
-    - Update the version in `package.json`. In the case of `iotivity-node`, 
-    the version becomes `1.2.0-3`. For versioning, we recommend following 
-    the pre-release version scheme as described by 
+    - Update the version in `package.json`. In the case of `iotivity-node`,
+    the version becomes `1.2.0-3`. For versioning, we recommend following
+    the pre-release version scheme as described by
     [semver.org](http://semver.org/#spec-item-9) e.g. `1.2.0-napi`.
     - Go through the release checklist (ensure tests/demos/docs are OK)
     - `npm publish --tag n-api`
 
-In this example, tagging the release with `n-api` has ensured that, although 
-version 1.2.0-3 is later than the non-N-API published version (1.2.0-2), it 
-will not be installed if someone chooses to install `iotivity-node` by simply 
-running `npm install iotivity-node`. This will install the non-N-API version 
-by default. The user will have to run `npm install iotivity-node@n-api` to 
-receive the N-API version. For more information on using tags with npm check 
+In this example, tagging the release with `n-api` has ensured that, although
+version 1.2.0-3 is later than the non-N-API published version (1.2.0-2), it
+will not be installed if someone chooses to install `iotivity-node` by simply
+running `npm install iotivity-node`. This will install the non-N-API version
+by default. The user will have to run `npm install iotivity-node@n-api` to
+receive the N-API version. For more information on using tags with npm check
 out ["Using dist-tags"][].
 
-## To introduce a dependency on an N-API version of a package
+# To introduce a dependency on an N-API version of a package
 
-To add the N-API version of `iotivity-node` as a dependency, the `package.json` 
+To add the N-API version of `iotivity-node` as a dependency, the `package.json`
 will look like this:
 
-```JSON
+```json
```

Comment:
For posterity `JSON` does not seem to work as expected.

![screen shot 2017-05-26 at 08 34 09](https://cloud.githubusercontent.com/assets/1443911/26483740/e009c21c-41ee-11e7-955f-bc1fbcb9a210.png)